images/miq-app/docker-assets/container-deploy-common.sh

```diff
@@ -67,6 +70,22 @@ fi
 
 }
 
+function check_version_gt() { 
+# Check if upgrade version is actually greater than stored PV version
+# -V sorts alphanumeric versions within text, will always return oldest version first
+# Compare sort version result against upgrade version
+
+if [[ "$(echo "$@" | tr " " "\n" | sort -V | head -n 1)" != "$1" ]]; then
```

Comment:
Ah I see. I missed the `$()` enclosing the string of commands. Sorry for the noise :smile: